src/main/java/operation/AdditionHandler.java

```diff
@@ -0,0 +1,15 @@
+package operation;
+
+import model.Record;
+import report.Report;
+
+public class AdditionHandler implements OperationHandler {
```

Comment:
Maybe better name it SupplyHandler, because the operation of addition is used twice, the same in other handlers, but it's not important.

Reply:
I think that addition is OK